tests/ord-aggregator/tests/handler_test.go

```diff
@@ -81,51 +79,38 @@ const (
-
-func init() {
-	rand.Seed(time.Now().UnixNano())
-}
-
-func randString(n int) string {
-	buffer := make([]rune, n)
-	for i := range buffer {
-		buffer[i] = letters[rand.Intn(len(letters))]
-	}
-	return string(buffer)
-}
-
 func TestORDAggregator(t *testing.T) {
 	basicORDConfigSecurity := &fixtures.ORDConfigSecurity{
-		Enabled:  true,
-		Username: randString(8),
-		Password: randString(8),
+		Username: testConfig.BasicUsername,
+		Password: testConfig.BasicPassword,
 	}
 
 	oauthORDConfigSecurity := &fixtures.ORDConfigSecurity{
-		Enabled:  true,
-		Username: randString(8),
-		Password: randString(8),
-		TokenURL: testConfig.ExternalServicesMockBaseURL + "/oauth/token",
+		Username: testConfig.ClientID,
+		Password: testConfig.ClientSecret,
+		TokenURL: testConfig.ExternalServicesMockBaseURL + "/secured/oauth/token",
 	}
 
-	toggleORDConfigSecurity(t, testConfig.ExternalServicesMockBaseURL+"/.well-known/open-resource-discovery/basic/configure", basicORDConfigSecurity)
-	toggleORDConfigSecurity(t, testConfig.ExternalServicesMockBaseURL+"/.well-known/open-resource-discovery/oauth/configure", oauthORDConfigSecurity)
+	accessStrategyConfigSecurity := &fixtures.ORDConfigSecurity{
+		AccessStrategy: "sap:cmp-mtls:v1",
+	}
 
-	var appInput, secondAppInput, thirdAppInput, fourthAppInput, fifthAppInput directorSchema.ApplicationRegisterInput
+	var appInput, secondAppInput, thirdAppInput, fourthAppInput, fifthAppInput, sixthAppInput directorSchema.ApplicationRegisterInput
 	t.Run("Verifying ORD Document to be valid", func(t *testing.T) {
-		appInput = fixtures.FixSampleApplicationRegisterInputWithORDWebhooks(expectedSystemInstanceName, expectedSystemInstanceDescription, testConfig.ExternalServicesMockAbsoluteURL, &fixtures.ORDConfigSecurity{})
-		secondAppInput = fixtures.FixSampleApplicationRegisterInputWithORDWebhooks(expectedSecondSystemInstanceName, expectedSecondSystemInstanceDescription, testConfig.ExternalServicesMockBaseURL, &fixtures.ORDConfigSecurity{})
-		thirdAppInput = fixtures.FixSampleApplicationRegisterInputWithORDWebhooks(expectedThirdSystemInstanceName, expectedThirdSystemInstanceDescription, testConfig.ExternalServicesMockBaseURL+"/basic/.well-known/open-resource-discovery", basicORDConfigSecurity)
-		fourthAppInput = fixtures.FixSampleApplicationRegisterInputWithORDWebhooks(expectedFourthSystemInstanceName, expectedFourthSystemInstanceDescription, testConfig.ExternalServicesMockBaseURL+"/oauth/.well-known/open-resource-discovery", oauthORDConfigSecurity)
-		fifthAppInput = fixtures.FixSampleApplicationRegisterInputWithORDWebhooks(expectedFifthSystemInstanceName, expectedFifthSystemInstanceDescription, testConfig.ExternalServicesMockBaseURL+"/cert/.well-known/open-resource-discovery", &fixtures.ORDConfigSecurity{})
+		appInput = fixtures.FixSampleApplicationRegisterInputWithORDWebhooks(expectedSystemInstanceName, expectedSystemInstanceDescription, testConfig.ExternalServicesMockAbsoluteURL, nil)
```

Comment:
would be useful if you add above each app a comment about the pair of auths tested with it